docs/concepts/public-utilities.md

```diff
@@ -34,6 +34,29 @@ From there, an internal system extracts the relevant information from the reques
 
 Your browser may have a local cache of the content in question and might not reflect that something has been blocked on the gateways. To avoid browser caching, attempt to view the content using your browser's incognito or private mode. You can also prevent caching issues by using a command-line tool such as Curl or Wget.
 
+### Supported Features
+
+The above public gateways support the following:
+
+- `ipfs.io` and `dweb.link` support the [full set of IPFS Gateway specifications](https://specs.ipfs.tech/http-gateways/).
+- `trustless-gateway.link` supports only the [Trustless Gateway subset of the specification](https://specs.ipfs.tech/http-gateways/trustless-gateway/).
+
+They support HTTP clients reaching them over both ipv4 and ipv6 addresses
+
+The underlying IPFS nodes backing the gateways support the following mutable identifiers under the `/ipns` namespace:
+- IPNS Public Keys, e.g. `ipfs.io/ipns/k51q....`
+- DNSLink for all IANA registered domains as well as `.crypto` and `.eth`, e.g. `vitalik-eth.ipns.dweb.link`
```

Comment:
Which example URLs should we be emphasizing? My feeling is dweb.link given it has origin isolation (particularly for web apps).

Should these be

```suggestion
- IPNS Public Keys, e.g. `dweb.link/ipns/k51q....`
- DNSLink for all IANA registered domains as well as `.crypto` and `.eth`, e.g. `dweb.link/ipns/vitalik.eth`
```

cc @2color @lidel

Reply:
I'd go with `dweb.link` + paths just because it's the most easily comprehensible (no dots converted to dashes)

Reply:
nit: IANA → ICANN
```suggestion
- IPNS Public Keys, e.g. `dweb.link/ipns/k51q....`
- DNSLink for all [ICANN](https://en.wikipedia.org/wiki/ICANN) registered domains as well as `.crypto` and `.eth`, e.g. `dweb.link/ipns/vitalik.eth`
```